README.rst

```diff
@@ -64,6 +64,22 @@ ACRN support systems:
   the Github issues system: https://github.com/projectacrn/acrn-hypervisor/issues.
   You can browse through the reported issues and submit issues of your own.
 
+* **Reporting a Potential Security Vulnerability**: If you have discovered potential
+  security vulnerability in ACRN, please send an e-mail to secure@intel.com. For issues
+  related to Intel Products, please visit https://security-center.intel.com.
+
+  It is important to include the following details:
+  - The projects and versions affected
```

Comment:
add a blank line before the first element of a list

Reply:
@dbkinder have updated, please review ,thanks.
-Yonghua